content/posts/status_update_08_24.md

```diff
@@ -0,0 +1,53 @@
+
+We will require at least one more beta to clean up some issues encountered between X11 and Wayland sessions, as well as take in some great fixes provided by community members.   But, it feels like we are almost there.
+
+## DebConf 24 
+
+<img src="/PXL_20240729_102150414.jpg" alt="Soumya and Ken at DebConf 24" title="Soumya and Ken at DebConf 24" width="320" style="float: right; margin: 35px;"/>
+
+Due to the generous support of our financial contributors, Soumya and I were able to meet up at [DebConf 24](https://debconf24.debconf.org/) this year in Busan, Korea.  This is our second in-person conference meet up; last year Canonical graciously sponsored both of us to attend the Ubuntu Summit in Riga, Latvia.  That event was incredibly productive and so we were both keen on further opportunities to collaborate in person and with the larger Debian world.
+
+The travel costs associated with our attendance at the conference was paid from our [Open Collective account](https://opencollective.com/regolith/donate).  While we do not spend a lot of effort to advertise or solicit financial contributions, happily there are enough people donating to the project to allow us to travel from time to time when advantageous to the project and it fits into our personal schedules. There are also a few financial contributors over the years that have contributed rather generous sums to the project.  While I don't feel it appropriate to call them out by name here, we are very happy to have received these funds to support the project.  
+
+We aim to invest all of the financial contributions back into the project such that users will receive value back in terms of better quality software, broader support, and healthy working relationships with related projects.
+
+## What's Coming Next
+
+### Debian Project Integration
+
+Soumya has [begun the effort](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1077305) to integrate the Wayland session into the Debian package repository.  Our goal is to integrate the entire wayland session into Debian.  Once there, we plan to then provide a "spin" or "flavor" to downstream Ubuntu.  I hope that this will eventually result in an officially supported flavor that will enable users to install Regolith from an ISO as has been done in past LTS releases.  However due to time constraints, I will not take on providing an ISO for the 24.04 release.
```

Comment:
We might also link to the actual package that made it into debian instead of the bugreport (https://packages.debian.org/sid/trawld)